services/movie_session.py

```diff
@@ -42,3 +42,10 @@ def update_movie_session(
 
 def delete_movie_session_by_id(session_id: int) -> None:
     MovieSession.objects.get(id=session_id).delete()
```

Comment:
The `get` method used here can raise a `DoesNotExist` exception if the `MovieSession` with the given `id` is not found. Consider handling this exception to ensure the function behaves as expected even if the session does not exist.